Makefile_packaging.mk

```diff
@@ -411,13 +423,13 @@ chrootbuild: $(SRPM) $(CALLING_MAKEFILE)
             add_repos+=" --repo $$baseurl";                                 \
     done;                                                                   \
 	distro_repos="";                                                    \
-	for repo in $($(basename $(DISTRO_ID))_LOCAL_REPOS)                 \
-	            $($(basename $(DISTRO_ID))_REPOS); do                   \
+	for repo_key in $(SUSE_REPO_KEYS); do                               \
+	    $(call install_gpg_key,$$repo_key);                              \
+	done;                                                               \
+	for repo in $($(DISTRO_BASE)_LOCAL_REPOS)                 \
+	            $($(DISTRO_ID_BASE)_REPOS); do                   \
```

Comment:
Why do these two lines have different variable substitutions in them when they are both replacing the same common variable?

Reply:
DISTRO_BASE is upper case distro name underscore major version to match the Jenkins environment variables.
DISTRO_ID_BASE is lowercase distro name with the major version appended to it to match the makefile additions.

Reply:
Hrm.  Do we have too many representations of a single thing here?  The lower-case uses (i.e. `el7`, `sle12`, etc.) in the project `Makefile`s can easily be converted to use the upper-case nomenclature.